src/ray/gcs/gcs_server/gcs_job_manager.cc

```diff
@@ -124,7 +124,14 @@ void GcsJobManager::HandleAddJob(rpc::AddJobRequest request,
       // multiple times and requires idempotency (i.e. due to retry).
       running_job_start_times_.insert({job_id, job_table_data.start_time()});
     }
+
+    // Send driver job event to export API.
+    // TODO(can-anyscale): to be deprecated once the event aggregator is stable.
     WriteDriverJobExportEvent(job_table_data);
```

Comment:
I would wrap this in a utility function and inside the function leave a comment explaining the context (that we're mid migration and `WriteDriverJobExportEvent` will be deprecated)

maybe `WriteDriverJobExportEvent` is already that utility function?